pkg/bootstrap/bootstrap_config.go

```diff
@@ -50,6 +50,21 @@ const (
 	IstioMetaJSONPrefix = "ISTIO_METAJSON_"
 
 	lightstepAccessTokenBase = "lightstep_access_token.txt"
+
+	// statsPatterns gives the developer control over Envoy stats collection
+	EnvoyStatsMatcherInclusionPatterns = "sidecar.istio.io/v1alpha1/statsPatterns"
```

Comment:
would something like: `sidecar.istio.io/v1alpha1/statsInclusionPrefixes` be clearer as to what to allow. We only support prefixes and only support inclusions with this change, correct?

Reply:
OK, I made that change.

Do you know of a way to regenerate the pkg/bootstrap/testdata/*_golden.json files?  I have been editing by hand.  `REFRESH_GOLDEN` isn't working for .json files in this directory.  I am concerned that I was supposed to do something to enable the golden file that I added for the new test of this feature to be generated.